features/nanostack/mbed-mesh-api/source/WisunInterface.cpp

```diff
@@ -470,6 +470,42 @@ mesh_error_t WisunInterface::validate_timing_parameters(uint16_t disc_trickle_im
     return MESH_ERROR_NONE;
 }
 
+mesh_error_t WisunInterface::set_device_min_sens(uint8_t device_min_sens)
+{
+    int status = ws_device_min_sens_set(get_interface_id(), device_min_sens);
+    if (status != 0) {
+        return MESH_ERROR_UNKNOWN;
+    }
+
+    return MESH_ERROR_NONE;
+}
+
+mesh_error_t WisunInterface::get_device_min_sens(uint8_t *device_min_sens)
+{
+    if (device_min_sens == NULL) {
+        return MESH_ERROR_PARAM;
+    }
+
+    ws_stack_info_t stack_info = {0};
+
+    if (ws_stack_info_get(get_interface_id(), &stack_info)) {
```

Comment:
We do not have any get api for device_min_sense. It comes along with the stack info. That is why I have implemented this way. 
While configuring the mesh interface, the NM calls this get_device_min_sens API to get the default value of the device min sense. But at that moment the interface is not connected. So this API returns failure. 
@mikter please review this API.

Reply:
There is no actual min sensitivity configuration get function. This is the value stack has calculated based on the packets it has received.

it is not a configuration what you are reading here it is the actual value that stack has calculated.

The default value for this is 0 meaning stack will try to find the sensitivity limit and if you read that value and write it again it disables the feature. This is not the intended functionality

I think you should return 0 or store the value in the wisun interface until we create a real getter for that configuration.
This stack info value should be visible in statistics and not configured

@mikaleppanen what is your take on this?

Reply:
Okay. I will simply return 0 from this API. 
Later we can modify this API when actual get API will be available.